src/utils/productUtils/filterProductsByCategory.jsx

```diff
@@ -0,0 +1,20 @@
+function filterProductsByCategory(products, category) {
```

Comment:
Really liked this approach. awesome work 🔥

Reply:
thanks